examples/keycloak/server.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs')
+const path = require('path')
+const express = require('express')
+const { makeExecutableSchema } = require('graphql-tools')
+
+const { ApolloVoyagerServer, gql } = require('../../packages/apollo-voyager-server')
```

Comment:
[Optional] We could use package name instead to support copy paste etc. 
Locally package will be linked so it should work fine